### PR TITLE
NAS-119245 / 23.10 / Adjust ix-select width if chosen element has long name

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Screen Shot 2022-12-05 at 10 14 31
Screen Shot 2022-12-05 at 10 14 43

As well fixed title here:
Before:
Screen Shot 2022-12-05 at 12 36 03

After:
Screen Shot 2022-12-05 at 12 34 35

### CODE REVIEW

**Comment by undsoft** [comment]
I would prefer to wrap text to next line as opposed to cutting. In this example I may have a long nested dataset path that I may still want to see.

**Comment by undsoft** [comment]
@AlexKarpov98 I think it would be okay without a limit.

**Comment by bvasilenko** [comment]
Works as expected:

image

image

**Review by bvasilenko — APPROVED** [review]

File: src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="header">
-  <h3 class="title">
+  <h3 class="title {{ !(!dataset.locked && !isZvol) ? 'no-buttons' : ''}}">
```
Comment (bvasilenko) [inline src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html:2]: I noticed that you've used inversion operator `!` twice. I'd suggested to simplify, and here's 2 options:
- `dataset.locked || isZvol`
- or `(!dataset.locked && !isZvol) ? '' : 'no-buttons'`

**Comment by AlexKarpov98** [comment]
@undsoft - please approve and merge if it's fine now :)

**Review by undsoft — APPROVED** [review]
Yes, looks good.

**Pull request merged**